src/Modules/Trinity.FFI/Trinity.FFI/Agent/Storage.CellAccessorManager.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections.Generic;
```

Comment:
I'm still wondering why do we favor pythonnet over our custom method thunks

Reply:
This is just the slow version for Python binding.  
I'm to implement a fast one with gasm, and then a new version totally written with c-extensions would be available.  
During the lasy two days I'm implementing gasm with cpp...

Reply:
The slow version is mainly for data transformation.  
When you have to load data from Graph Engine servers, just use current implementation and you will get original python obejcts.